conf/workloads/ml/classification.conf

```diff
@@ -0,0 +1,42 @@
+hibench.classification.tiny.examples                    100
+hibench.classification.tiny.features                    100
+hibench.classification.tiny.frac_categorical_features   0
+hibench.classification.tiny.frac_binary_features        0
+hibench.classification.small.examples                   1000
+hibench.classification.small.features                   1000
+hibench.classification.small.frac_categorical_features  0
+hibench.classification.small.frac_binary_features       0
+hibench.classification.large.examples                   10000
+hibench.classification.large.features                   10000
+hibench.classification.large.frac_categorical_features  0
+hibench.classification.large.frac_binary_features       0
+hibench.classification.huge.examples                    500000
+hibench.classification.huge.features                    100000
+hibench.classification.huge.frac_categorical_features   0
+hibench.classification.huge.frac_binary_features        0
+hibench.classification.gigantic.examples                    1000000
+hibench.classification.gigantic.features                    1000000
+hibench.classification.gigantic.frac_categorical_features   0
+hibench.classification.gigantic.frac_binary_features        0
+hibench.classification.bigdata.examples                    2000000
+hibench.classification.bigdata.features                    10000000
+hibench.classification.bigdata.frac_categorical_features   0
+hibench.classification.bigdata.frac_binary_features        0
+
+
+hibench.classification.examples			${hibench.classification.${hibench.scale.profile}.examples}
+hibench.classification.features			${hibench.classification.${hibench.scale.profile}.features}
+hibench.classification.frac_categorical_features	${hibench.classification.${hibench.scale.profile}.frac_categorical_features}
+hibench.classification.frac_binary_features	${hibench.classification.${hibench.scale.profile}.frac_binary_features}
+hibench.classification.labeltype                  2
+hibench.classification.tree_depth			1
+
+
+hibench.classification.partitions		${hibench.default.map.parallelism}
+# storageLevel for the RDDs 0: off-heap 1: disk_only 3: memory_only  5: memory_ser 7: memory_and_disk 9: memory_and_disk_ser
+hibench.classification.storage_level           7
+hibench.classification.disable_kryo            false
```

Comment:
Are these copied from Nweight's conf file? Remove them if they are not used in this workload.